src/index.html

```diff
@@ -6,7 +6,37 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
-  </body>
-</html>
+  <body class="month">
+    <div class="month__day day-1"></div>
```

Comment:
```suggestion
    <div class="month__day"></div>
```
leave only `month__day` class for all divs

Reply:
I don't know how add a working demo link. Before this task everything was working.